geoutils/georaster.py

```diff
@@ -43,13 +43,13 @@ class Raster(object):
     _is_modified = None
     _disk_hash = None
 
-    def __init__(self, filename, bands=None, load_data=True, downsampl=1,
```

Comment:
Just since wer're on revising attribute names, is `downsampl` an abbreviation of `downsample`? Feels like an abbreviation just for abbreviates' sake!

Reply:
Agree, changed to `downsample`.